<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxtMsgEncryption</class>
 <widget class="QMainWindow" name="TxtMsgEncryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encrypt Text Message</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="ptePlainTextMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type yout message here...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEncrypt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEncryptedMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypted message:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteEncryptedMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblKey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteKey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSend">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
